sys/hacl/build.rs

```diff
@@ -25,34 +30,42 @@ fn includes(home_dir: &Path) -> Vec<String> {
 }
 
 fn append_simd128_flags(flags: &mut Vec<String>) {
-    // Platform detection
-    if cfg!(simd128) {
-        flags.push("-DSIMD128".to_string());
-        if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
-            flags.push("-msse4.1".to_string());
-            flags.push("-msse4.2".to_string());
-            flags.push("-mavx".to_string());
-        }
+    if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
+        flags.push("-msse4.1".to_string());
+        flags.push("-msse4.2".to_string());
+        flags.push("-mavx".to_string());
     }
 }
 
 fn append_simd256_flags(flags: &mut Vec<String>) {
-    // Platform detection
-    if cfg!(simd256) {
-        flags.push("-DSIMD256".to_string());
-        flags.push("-mavx2".to_string());
-    }
+    flags.push("-mavx2".to_string());
 }
 
 #[cfg(not(windows))]
-fn create_bindings(home_dir: &Path) {
-    let mut clang_args = includes(home_dir);
-    append_simd128_flags(&mut clang_args);
-    append_simd256_flags(&mut clang_args);
+fn create_bindings(platform: Platform, home_dir: &Path) {
```

Comment:
The Windows function has a different function signature now and will fail.
Ideally we don't need this guard, but I think Windows has some issues here.
And since we want to move away from creating bindings every time, let's not try fix this for now. But it should build on Windows too.

Reply:
Agreed. We can make `create_bindings` work on Windows at later point.